### PR TITLE
✨ `document-issue-note` output format

### PR DESCRIPTION
**Pull request by ollyhensby**
- Support new `document-issue-note` format that allows users to print without all the document-issue title page, contents page, and table of contents.
- Support landscape orientation, and A3 paper size for both output formats.

### CODE REVIEW

File: packages/document-issue-io/src/document_issue_io/markdown_document_issue.py [hunk 1]
```diff
@@ -115,10 +119,28 @@ def check_quarto_version() -> ty.Union[None, str]:
...
+class PaperSize(Enum):
+    A4 = "a4"
+    A3 = "a3"
+
+
 def run_quarto(
-    fpth_md_output: pathlib.Path, fpth_pdf: pathlib.Path
+    fpth_md_output: pathlib.Path,
+    fpth_pdf: pathlib.Path,
+    output_format: OutputFormat = OutputFormat.DOCUMENT_ISSUE_REPORT,
```
Comment (jgunstone) [inline packages/document-issue-io/src/document_issue_io/markdown_document_issue.py:140]: move output format to yaml

File: packages/document-issue-io/src/document_issue_io/markdown_document_issue.py [hunk 2]
```diff
@@ -146,19 +168,60 @@ def run_quarto(
...
+                document_issue=document_issue,
+                fpth_output=fpth_output,
+                is_titlepage=is_title_page,
+            )
+        else:
+            raise ValueError(
+                "Other paper sizes and orientations are not supported at this time."
+            )
+        # Create quarto yaml defining the orientation and paper size
+        import yaml
```
Comment (jgunstone) [inline packages/document-issue-io/src/document_issue_io/markdown_document_issue.py:212]: fix imports

**Review by jgunstone — APPROVED** [review]
looks good other than the few minor comments

File: packages/document-issue-io/README.md [hunk 3]
```diff
@@ -77,6 +77,12 @@ the `dist` directory.
 
 The hatch build hook that packages the Quarto extensions is defined in the [hatch_build.py](hatch_build.py) file.
 
+## Updating Quarto extension
+
+To pack `document-issue-quarto` and use it in this repo, run the following command (from root folder):
+
+```tar -czvf packages/document-issue-io/src/document_issue_io/templates/document-issue-quarto.tar.gz -C packages/document-issue-quarto --transform 's,^_extensions,_extensions,' _extensions```
+
 ## License
```
[the remark this answers is not in the document]
Comment (ollyhensby) [reply packages/document-issue-io/README.md:85]: Done. Created `pixi run update-quarto-extensions`.

File: packages/document-issue-io/src/document_issue_io/markdown_document_issue.py [hunk 4]
```diff
@@ -115,10 +119,28 @@ def check_quarto_version() -> ty.Union[None, str]:
         return quarto_version
 
 
+class OutputFormat(Enum):
```
[the remark this answers is not in the document]
Comment (ollyhensby) [reply packages/document-issue-io/src/document_issue_io/markdown_document_issue.py:122]: Done.

File: packages/document-issue-io/src/document_issue_io/markdown_document_issue.py (hunk 1 above)
Comment (ollyhensby) [reply packages/document-issue-io/src/document_issue_io/markdown_document_issue.py:140]: Same comment as above. Have done this.

**Pull request merged**